app/models/concerns/thredded/content_moderation_state.rb

```diff
@@ -30,16 +30,16 @@ module ContentModerationState
       end)
     end
 
-    def visible_to_all?
+    def moderation_state_visible_to_all?
```

Comment:
@glebm I found this a bit confusing at first (I read it as "is Moderation State (whether moderated or not)visible to all / user" 

Of course it really means "is it (due to moderation state) visible to user". It would be nice but I don't think it's possible to crisply pack that meaning into the method names. Maybe comment(s) such as following might help a future me:

```ruby
  #determines if this should be visible to anyone based on the moderation state
  def moderation_state_visible_to_all?
...
  #determines if this should be visible to a specific user based on the moderation state`
  def moderation_state_visible_to_user?(user)
```